src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class name `calendar--month-length-31` is too specific and not scalable. Consider using a more generic naming convention that can handle different month lengths dynamically with JavaScript.

Reply:
Estou utilizando as ferramentas aprendidas até o momento